flow360/component/flow360_params/solvers.py

```diff
@@ -185,74 +240,58 @@ class TurbulenceModelSolver(GenericFlowSolverSettings):
 
     Example
     -------
-    >>> ts = TurbulenceModelSolver(model_type='SA', absolute_tolerance=1e-10)
+    >>> ts = TurbulenceModelSolver(absolute_tolerance=1e-10)
     """
 
-    model_type: TurbulenceModelModelType = pd.Field(
-        alias="modelType", default=TurbulenceModelModelType.SA
+    model_type: str = pd.Field(alias="modelType")
```

Comment:
Use Literal for the model type as this is not abstract class

Reply:
Fixed in c3c40f4